### PR TITLE
Child theme footer

### PR DESCRIPTION
**Pull request by joshdarby**
## Changes

This pull request makes the following changes:

- Creates relevant template files + styles for Workday child theme footer to match design. Does not include logos section.

Screen Shot 2019-07-31 at 3 43 38 PM

## Why

For #3 

## Testing/Questions

Features that this PR affects:

- Workday child theme footer

Steps to test this PR:

1. Add in social links to theme options
2. Verify the footer styling looks ok

### CODE REVIEW

**Comment by benlk** [comment]
Facebook: https://www.facebook.com/WorkdayMinnesota
Twitter: https://twitter.com/workdaymn
YouTube: https://www.youtube.com/channel/UCiZUXe78OaSxdJmUfO6l-_w

File: wp-content/themes/workday/less/inc/footer.less [hunk 1]
```diff
@@ -0,0 +1,53 @@
+.footer-bg {
+    background: @black;
+    #site-footer {
+        #boilerplate {
+            padding-top: 50px;
+            padding-bottom: 15px;
+            border-top: 2px solid @gray;
+            font-family: @font-body;
```
Comment (benlk) [inline wp-content/themes/workday/less/inc/footer.less]: should be `@font-headline` based on the mockup in https://github.com/INN/umbrella-workdayminn/issues/3, I think?

But honestly, `@font-body` looks better here and let's roll with it.

File: wp-content/themes/workday/partials/footer-widget-1col.php [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<div class="span12 widget-area" role="complementary">
+	<!-- placeholder for logos -->
+</div>
```
Comment (benlk) [inline wp-content/themes/workday/partials/footer-widget-1col.php]: Is creating this file necessary if we're going to be creating a widget to hold the logos? If we're just creating a widget, can we use the unmodified https://github.com/INN/largo/blob/v0.6.4/partials/footer-widget-1col.php ?

Comment (joshdarby) [reply wp-content/themes/workday/partials/footer-widget-1col.php]: I wasn't sure what our plan was for the logos so I just created this placeholder for them / so I could get rid of the supplementary menu that was appearing.

Comment (benlk) [reply wp-content/themes/workday/partials/footer-widget-1col.php]: Let's not worry about it until we have a plan for the logos.

Comment (joshdarby) [reply wp-content/themes/workday/partials/footer-widget-1col.php]: Alright. I removed the file in https://github.com/INN/umbrella-workdayminn/pull/20/commits/dbf355380e5b32306b1baedcd28f195515962217.

**Pull request merged**